packages/@okta/vuepress-theme-prose/global-components/ErrorCodes.vue

```diff
@@ -299,7 +299,7 @@ export default {
 
   .error-codes .error-code h4 {
     clear: left;
-    margin: 25px 0 0;
+    margin: 32px 0 0;
```

Comment:
I thought you wouldn't need to change styles here because we have margin 25px + padding 6px and it's very close to 32px we have for all other anchors.
By testing the preview link I see it works fine

Reply:
When I click on a link icon
<img width="432" alt="image" src="https://github.com/okta/okta-developer-docs/assets/4158731/434fc1c1-8bd9-42ed-bb55-c8edb2763990">

When I use the address bar
<img width="405" alt="image" src="https://github.com/okta/okta-developer-docs/assets/4158731/53e477b2-cbb5-4a86-a9a8-3a4f5fb1cb1f">

For the first option, we have that padding 6px. We can ignore this one as it's a very minor thing